src/Api/ItemList.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace OpsWay\ZohoBooks\Api;
+
+
+class ItemList implements \ArrayAccess, \IteratorAggregate
+{
+    /**
+     * @var array
+     */
+    private $items = [];
+
+    /**
+     * @var array
+     */
+    private $pageContext = [];
+
+    function __construct($items, $pageContext)
```

Comment:
Missing visibility declaration. BTW, could you please keep the type hinting for these arguments?